react/features/base/config/configWhitelist.ts

```diff
@@ -81,6 +81,7 @@ export default [
     'brandingRoomAlias',
     'debug',
     'debugAudioLevels',
+    'deeplinking',
```

Comment:
I don't think we allowed overriding of the app links did we?

Reply:
I think not, but it seems to be a flaw, as if an integrator wants to configure deeplink to another app, they cannot set all their own app data if we restrict part of the settings.

Reply:
That should be done via branding. Otherwise someone could send someone else to a malicious app with a custom crafted link.

Reply:
Now I only allow previously allowed settings to be overwritten.